install-local.sh

```diff
@@ -144,6 +167,28 @@ ns_timestamp=$(date +%Y-%m-%ST%H:%M:%S%z | sed 's/[0-9][0-9]$/:&/')
 echo "$ns_timestamp" > "$ns_build_path/timestamp"
 echo "${ns_sysname:=$(hostname -s)}" > "$ns_build_path/sysname"
 
+# Set up python virtual environment.
+
+if [ "$ns_pyvenv" != disable ]; then
+    echo
+    msg "Initializing python virtual environment"
+    ns_pyvenv_opt=
+    if [ "$ns_pyvenv" == inherit ]; then
+	ns_pyvenv_opt=--system-site-packages
+    fi
+
+    msg "Installing python modules: $ns_pyvenv_modules"
+    (
+	exec >> "$ns_build_path/log_pyvenv" 2>&1
+	if python3 -m venv $ns_pyvenv_opt "$ns_pyvenv_path"; then
+	    source "$ns_pyvenv_path/bin/activate"
+	    for pkg in $ns_pyvenv_modules; do
+	        pip install "$pkg"
```

Comment:
On OS X with homebrew you need to use `pip3` explicitly. Again, we might need a`ns_pip` variable, with some testing in the environment setup to find pip for Python 3.

Reply:
Can I confirm with you that that still holds within the virtual environment? In principle, it should automatically 'install' pip in the environment, and I'm hoping that that means that `pip` alone will work.